### ISSUE TITLE
[Feature]: Sorting albums and album groups

### ISSUE THREAD

**Issue by PeterBasista** [immich-app/immich#1133]
### Feature detail

Hey, guys!
 
I could really use a few things for the albums:
- sorting albums (albums page) by date and alphabetically - the current sorting is by creation date. The new one would be according to what the user chooses. E.g. date - it will show the albums where the most recent photos are first and then the older ones. This logic would work based on selecting the newest photo in a given album, this would also present the main element for sorting that album against the others (also reverse sorting, from oldest).
- Option to create a group of albums, this group would collect the selected album. If you go to the mountains often but to the seaside a little on holidays, it is consequently harder to find among the albums the ones that were when you were at the seaside. Using groups, one could sort the albums (user-defined group names) into "holidays by the sea" and "trips to the mountains". Maybe using tags for albums would solve this problem and thus tags would be added to the previous sorting by time/alphabet, after which only albums with the given tag would be displayed (in this case it would be a multiselect - i.e. sorting by date asc where the tag is "xyz")
- displaying the "favorites" section at the top of the album. Photos that are tagged as favorites could be displayed at the top as a favorites section for that album. This section would be followed by a list of all photos in that album as before (including photos that are in the favorites section).
- only show assets in the album that are marked as visibility: true (in this case I don't know if this is a bug or a feature). But after importing the library I created albums and stacked photos, then I started to solve the photo-video linking problem for apple livephotos. After successfully updating the database, I browsed the albums and noticed that they contain already livephotos but also originally videos, despite being marked as visibility: false in the DB (if interested I can offer sql query to sync livephoto photos and videos)

### Platform

Web

**Comment by jrasm91** [comment]
You manually linked previously uploaded photos and videos?

I would be interested in seeing how you did that.

**Comment by PeterBasista** [comment]
Yes. Basically, every video and photo that is like a live photo is created at the same time (+-3 s) and has the same name. Later, I will edit this comment and add SQL that will update the previously uploaded livephotos. I'm not at my laptop right now.

**Comment by bo0tzz** [comment]
Partial duplicate of #385

**Comment by matthinc** [comment]
Sorting albums alphabetically and favorites (mobile) were introduced in https://github.com/immich-app/immich/pull/1501 and will be included in the next release.
